src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,341 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        printWelcomeMessage();
+        ArrayList<Task> tasks = new ArrayList<>();
+        String pathOfDataFile = loadListFromFile(tasks);
+        String command;
+        boolean isFinished = false;
+        Scanner input = new Scanner(System.in);
+        while (!isFinished && input.hasNextLine()) {
+            command = input.nextLine();
+            printSingleLine();
+            if (command.equals("bye")) {
+                printGoodbyeMessage();
+                isFinished = true;
+            } else if (command.equals("list")) {
+                printListOfTask(tasks);
+            } else {
+                updateListOfTask(command, tasks);
+                try {
+                    saveListToFile(tasks, pathOfDataFile);
+                } catch (IOException e) {
+                    System.out.println("I/O error has occurred");
+                }
+            }
+        }
+    }
+
+    private static String loadListFromFile(ArrayList<Task> tasks) {
+        checkIfDirectoryExists();
+        String pathOfDataFile = "data/duke.txt";
+        try {
+            File dataFile = checkIfFileExists(pathOfDataFile);
+            readListFromFile(tasks, dataFile);
+        } catch (IOException e) {
+            System.out.println("I/O error has occurred");
+        }
+
+        if (tasks.size() > 0) {
+            printListOfTask(tasks);
+        }
+        return pathOfDataFile;
```

Comment:
Why does `loadListFromFile` return the file path?

Reply:
I realized it is redundant. Have removed it.